packages/components/src/DataList/components/DataListItemActions/DataListItemActionsOverflow.tsx

```diff
@@ -33,7 +33,7 @@ export function DataListItemActionsOverflow<T extends DataListObject>({
         />
       </Tooltip>
 
-      {item && (
+      {actions && (
```

Comment:
@darryltec do you see an issue with this? I changed it because bulk actions will not have a specific item to pass here.

Reply:
nope!